.github/CHANGELOG.md

```diff
@@ -11,7 +11,8 @@
 10. AIRAC (2407) - Updated Land's End (EGHC) hold points - thanks to @RedstonePilot (Ben Walker)
 11. Bug - Corrected A-NorthWest Region naming discrepancy in GB Landmass - thanks to @matthewmoy
 12. AIRAC (2407) - Added new run up areas to the Redhill (EGKR) SMR - thanks to @kristiankunc (Kristián Kunc)
-13. AIRAC (2406) - Updated Barkston Heath (EGYE) runway headings - thanks to @mid-archie 
+13. AIRAC (2406) - Updated Barkston Heath (EGYE) runway headings - thanks to @mid-archie
+14. AIRAC (2407) - Updated Cambridge (EGSC) Radar Frequencies - Thanks to @mid-archie
```

Comment:
```suggestion
14. Procedure Change (2407) - Updated Cambridge (EGSC) Radar Frequencies - Thanks to @mid-archie
```
Feel free to ignore this one if not relevant, but I _think_ it's a procedure change rather than AIRAC change?

Reply:
Agreed